### DISCUSSION TITLE
Bots should not rack empty ammo boxes and instead drop them near desconstructor

### DISCUSSION THREAD

**Discussion by marpul358** [FakeFishGames/Barotrauma#9151] [Ideas and suggestions]
Bots should not place empty ammunition boxes in the ammunition box racks. I'm not sure which bot command it is but I'm sure it is at least the "load ammo" command. Would also be a neat addition if the bots with a load ammo command would drop empty ammunition boxes near the deconstructor.

Reason is that If you are in a single player game keeping ammo stocked near the guns becomes increasingly tedious when bots fill up the ammo box racks with empty ammo boxes even if there are non-empty boxes not racked.

**Comment by ekqvist** [comment]
> Bots should not place empty ammunition boxes in the ammunition box racks.

Thanks for the suggestion. This is something that has already been on our list of things to address.

**Comment by Scuba-Recon** [comment]
Was going to make a similar suggestion in that a tag could be made for "emptyammocontainer" used for coilgun/ammo racks that could be placed near fabricator/deconstructor for bots to relocate specific empty ammo box types rather than placing them among full boxes.
